### PR TITLE
[ISSUE] Add Navbar in Leaderboard

### LINKED ISSUE #46: Add Navbar in Leaderboard
Task,
1. Add navbar in Leaderboard

image

### PR DESCRIPTION
**Pull request by SpandanNepal**
closes #46 

Screenshot 2023-10-09 at 9 40 38 PM

### CODE REVIEW

File: assets/index.html [hunk 1]
```diff
@@ -8,14 +8,22 @@
...
-        alt="Leapfrog Open Source Logo"
-      />
-      <h1>Frogtoberfest Leaderboard 2023</h1>
-      <div class="leaderboard__table">{table}</div>
+    <div>
+      <nav>
+        <img src="../assets/LF_Opensource.svg" alt="Logo">
+        <div class="nav-links">
+          <a href="https://frogtober.lftechnology.com/#participation">Participation</a>
+          <a href="https://frogtoberfest-leaderboard.lftechnology.com/">Leaderboard</a>
```
Comment (Swechhya) [inline assets/index.html:16]: Do we need `Leaderboard`  as we would be in Leaderboard?

File: assets/style.css [hunk 2]
```diff
@@ -1,3 +1,4 @@
+@import "./nav.css";
```
Comment (mesaugat) [inline assets/style.css:1]: I think we can keep all the styles under a single file. This is a very simple webpage. No need for modularization at the moment.

File: assets/nav.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  position: relative;
+  left: 100%;
+  top: -25px;
+}
+
+@media screen and (max-width: 1100px) {
+  .nav-links {
+    display: none;
+  }
+}
```
Comment (mesaugat) [inline assets/nav.css:33]: @SpandanNepal Missing new line feed but doesn't matter. Check my comment below.

File: assets/index.html [hunk 4]
```diff
@@ -8,14 +8,21 @@
...
-        src="../assets/LF_Opensource.svg"
-        alt="Leapfrog Open Source Logo"
-      />
-      <h1>Frogtoberfest Leaderboard 2023</h1>
-      <div class="leaderboard__table">{table}</div>
+    <div>
+      <nav>
+        <img src="../assets/LF_Opensource.svg" alt="Logo">
+        <div class="nav-links">
+          <a href="https://frogtober.lftechnology.com/#participation">Participation</a>
```
Comment (mesaugat) [inline assets/index.html:15]: Can we use the frogtoberfest URL please? The other one frogtober.lftechnology.com is likely an alias.

```suggestion
          <a href="https://frogtoberbest.lftechnology.com/#participation">Participation</a>
```

File: assets/index.html [hunk 5]
```diff
@@ -8,14 +8,21 @@
...
+          <a href="https://frogtober.lftechnology.com/checker">Checker</a>
+          <a href="https://frogtober.lftechnology.com/">Events</a>
```
Comment (mesaugat) [inline assets/index.html:17]: We don't need events.

File: assets/index.html [hunk 6]
```diff
@@ -8,14 +8,21 @@
...
-        alt="Leapfrog Open Source Logo"
-      />
-      <h1>Frogtoberfest Leaderboard 2023</h1>
-      <div class="leaderboard__table">{table}</div>
+    <div>
+      <nav>
+        <img src="../assets/LF_Opensource.svg" alt="Logo">
+        <div class="nav-links">
+          <a href="https://frogtober.lftechnology.com/#participation">Participation</a>
+          <a href="https://frogtober.lftechnology.com/checker">Checker</a>
```
Comment (mesaugat) [inline assets/index.html:16]: @SpandanNepal Please move the checker as the last item on the nav. Take reference from https://frogtoberfest.lftechnology.com

**Comment by SpandanNepal** [comment]
@mesaugat PR comments have been resolved, could you please review?

**Review by Swechhya — APPROVED** [review]

**Pull request merged**